cmake/FreecivInstall.cmake

```
@@ -79,14 +79,14 @@ if(MSYS OR MINGW)
     # Function to analyze the third party dll files linked to the exe's
     #   Uses the repurposed variable from above to tell the function where
     #   the dll files are located. Ignores dll's that come with Windows.
-    file(GET_RUNTIME_DEPENDENCIES
+    file(GLOB exes "${CMAKE_INSTALL_PREFIX}/freeciv21-*.exe")
+	file(GET_RUNTIME_DEPENDENCIES
```

Just a pedantic comment :sweat_smile: 
```suggestion
	  file(GET_RUNTIME_DEPENDENCIES
```